src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -32,7 +32,30 @@ limitations under the License.
   <struct name="AttributeValuePair">
```

Comment:
This name does not match the spec.

Please go through this XML and make it match the spec.

Please also speak with @naveenkommareddi and @robszewczyk about whether it's OK to make the changes here and to the cert tests after SVE and whatnot.

Reply:
Scenes going back to provisional means these changes can be made.